django/otto/views.py

```diff
@@ -670,9 +679,83 @@ def cost_dashboard(request):
     pilot = request.GET.get("pilot", "all")
     feature = request.GET.get("feature", "all")
     cost_type = request.GET.get("cost_type", "all")
+    date_group = request.GET.get("date_group", "last_30_days")
+    start_date = request.GET.get("start_date", None)
+    end_date = request.GET.get("end_date", None)
 
-    # First, filter the costs by the selected options
     raw_costs = Cost.objects.all()
+
+    # Filter by dates
+    if date_group == "last_90_days":
+        start_date = timezone.now().date() - timedelta(days=90)
+        end_date = timezone.now().date() - timedelta(days=1)
```

Comment:
What's the rationale behind excluding the current date from the "last N days" date groups?

Reply:
The current day is incomplete. So it gives you a slightly false idea of what "a week" costs.

Reply:
Fair enough!